### PR TITLE
Fix #77 engagement edits

### LINKED ISSUE #77: Engagement: Edits
- [x] Final sentence reads "Problem Structuring Methods, such as the Strategic Choice Approach, Rich Pictures and Systems Thinking can aid these discussions." Add hyperlink to the methods to https://www.gov.uk/government/publications/systems-thinking-for-civil-servants/toolkit
- [x] Do we want to add something around if engagement and scoping suggest that AI/ML will be used, the analyst and commissioner need to engage around the ethical considerations?
- [x] How do engagement and scoping differ for multi-use models
- [x] #111 Typo in "If the main purpose of analysis is to cut out irrelevance and to simplify, it is essential for to identify what is relevant for a piece of analysis." 
- [x] 6.2 6th and 7th bullets should start with 'They'
- [ ] Fix assurer and approver responsibilities that are identical to Design section

### PR DESCRIPTION
**Pull request by irisoren-sg**

### CODE REVIEW

**Comment by irisoren-sg** [comment]
@Hurstharrier I have restructured the section to follow structure of other Part 2 chapters

* Intro 
   * Responsibilities by role
* Assurance of  the stage
* Multi-use models and stage
* Black-box models and stage

I feel that the black-box models section is not informative as written, and would appreciate comments. I was trying to pay attention to what would go in this stage vs the design stage

**Comment by irisoren-sg** [comment]
Also for multi-use models: "In the case of multi-use models, the Analyst may be required to engage with a group of end-users to develop an understanding of their respective requirements. As requirements might differ or contradict, techniques such as Strategic Options Development and Analysis ([SODA](https://www.ru.nl/publish/pages/938444/soda_-_the_principles.pdf)) and [Soft Systems Methodology]**ADD HYPERLINK** may be used to develop a shared understanding across multiple groups." 

I have never used SSM but from reading about it, it seems to do the same as SODA so included it. It would be good to get thoughts on whether this is correct as written, and if so, add a suitable reference

File: engagement_and_scoping.qmd [hunk 1]
```diff
@@ -7,34 +7,54 @@ The draft currently has no official status. It is a work in progress and is subj
...
 * They are available to actively engage with the Analyst to appropriately shape the work.    
 * There is sufficient governance in place to support the analysis and its role in the wider project or programme. This is particularly important if the analysis supports business critical decisions. This proportionality may need to be revisited at the design stage if a novel or riskier approach is required (for example if AI models are used).    
 * They understand risks where time and resource pressures constrain the approach.    
-* They request information on uncertainty from analysts and challenge them when it is absent, inadequate or ambiguous.    
-* Communicate to the analyst any sources of uncertainty they have identified as part of their wider considerations.    
-* If possible, indicate in advance the consequences for decision-making of different degrees of uncertainty, as this may enable the analyst to conduct their analysis at a proportionate level.    
-* Commissioner should sign off on the specification
+* They request information from the Analyst on accounting for uncertainty, and challenge them when it is absent, inadequate or ambiguous.
```
Comment (Hurstharrier) [inline engagement_and_scoping.qmd]: In my experience the commissioners seldom have much of an idea about sources of uncertainty or acceptable levels of uncertainty.

Might need to make this explicit.

**Review by Hurstharrier — COMMENTED** [review]
Have made comments and some edits

**Pull request merged**